packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -416,6 +417,7 @@ AnalyticalTable.defaultProps = {
   filterable: false,
   groupable: false,
   selectionMode: TableSelectionMode.NONE,
+  selectionBehavior: TableSelectionBehavior.ROW_SELECTOR,
```

Comment:
```suggestion
  selectionBehavior: TableSelectionBehavior. ROW,
```
Row is also the default now, so I would keep that behaviour.